docker-for-windows/faqs.md

```diff
@@ -111,62 +95,55 @@ learn more about running diagnostics and about Docker Desktop for Windows GitHub
+contains discussion threads. You can also create discussion topics there,
 but we recommend using the GitHub issues over the forums for better tracking and
 response.
 
 ### How can I opt out of sending my usage data?
 
-If you do not want auto-send of usage data, use the Stable channel. For more
-information, see [Stable and Edge channels](#questions-about-stable-and-edge-channels) ("What is the difference between the Stable and Edge versions of Docker Desktop for Windows?").
+If you do not want to send usage data, use the Stable channel. For more
+information, see [Stable and Edge channels](#questions-about-stable-and-edge-channels) ("What is the difference between the Stable and Edge versions of Docker Desktop?").
 
 ### How is personal data handled in Docker Desktop?
 
 When uploading diagnostics to help Docker with investigating issues, the
 uploaded diagnostics bundle may contain personal data such as usernames and IP
 addresses. The diagnostics bundles are only accessible to Docker Inc. employees
-who are directly involved in diagnosing Docker Desktop issues. By default Docker
-Inc. will delete uploaded diagnostics bundles after 30 days unless they are
-referenced in an open issue on the
+who are directly involved in diagnosing Docker Desktop issues. By default, Docker Inc. will delete uploaded diagnostics bundles after 30 days unless they are referenced in an open issue on the
 [docker/for-mac](https://github.com/docker/for-mac/issues) or
 [docker/for-win](https://github.com/docker/for-win/issues) issue trackers. If an
 issue is closed, Docker Inc. will remove the referenced diagnostics bundles
 within 30 days. You may also request the removal of a diagnostics bundle by
-either specifying the diagnostics ID or via your GitHub ID (if the diagnostics
-ID is mentioned in a GitHub issue). Docker Inc. will only use the data in the
-diagnostics bundle to investigate specific user issues, but may derive high
-level (non personal) metrics such as the rate of issues from it.
+either specifying the diagnostics ID or through your GitHub ID (if the diagnostics ID is mentioned in a GitHub issue). Docker Inc. will only use the data in the diagnostics bundle to investigate specific user issues, but may derive high level (non-personal) metrics such as the rate of issues from it.
 
-### Can I use Docker Desktop for Windows with new swarm mode?
+### Can I use Docker Desktop with the Swarm mode?
 
-Yes! You can use Docker Desktop for Windows to test single-node features of
+Yes! You can use Docker Desktop to test single-node features of
 [swarm mode](/engine/swarm/) introduced with Docker Engine 1.12, including
 initializing a swarm with a single node, creating services, and scaling
 services. Docker “Moby” on Hyper-V serves as the single swarm node. You can also
-use Docker Machine, which comes with Docker Desktop for Windows, to create and
+use Docker Machine, which is bundled in Docker Desktop, to create and
```

Comment:
Should this be "bundled with?" Maybe we could say "...use Docker Machine, which comes installed in Docker Desktop," or "...use Docker Machine, which is preinstalled in Docker Desktop?" Would this change the technical meaning?

Reply:
Updated to 'bundled with'. 

I believe it is safe to say `bundled with` than preinstalled as Docker machine isn't technically installed until users complete the Desktop installation process.

Also, a list of components that are included in the default Desktop installation is documented in the Install topic.

Reply:
Updated in commit 59aa7e5.